### ISSUE TITLE
EL 7.7 Breaks `test_ami.py`

### ISSUE THREAD

**Issue by ferricoxide** [plus3it/spel#326]
### Expected behavior

Validator tests handle testing for Python v3.x appropriately

### Actual behavior

Validator fails because `yum` "magically" changes the Python v3.x installation from EPEL's `python36` package to the base repos' `python3` RPM. Found in the logs:

~~~
            minimal-rhel-7-hvm: Package python36 is obsoleted by python3, trying to install python3-3.6.8-10.el7.x86_64 instead
            minimal-rhel-7-hvm: Resolving Dependencies
            minimal-rhel-7-hvm: --> Running transaction check
            minimal-rhel-7-hvm: ---> Package python3.x86_64 0:3.6.8-10.el7 will be installed
~~~

### Steps to reproduce behavior

Run `spel` using any bootstrap AMI that gets updated to 7.7 (or, presumably, later): AMI is produced but gets nuked due to `python36` RPM-presence test:

~~~
            minimal-rhel-7-hvm: =================================== FAILURES ===================================
            minimal-rhel-7-hvm: ____________________ test_python3_installed[local-python36] ____________________
            minimal-rhel-7-hvm:
            minimal-rhel-7-hvm: host = <testinfra.host.Host object at 0x7fb8cb290250>, name = 'python36'
            minimal-rhel-7-hvm:
            minimal-rhel-7-hvm:     @pytest.mark.parametrize("name", [
            minimal-rhel-7-hvm:         ("python36")
            minimal-rhel-7-hvm:     ])
            minimal-rhel-7-hvm:     def test_python3_installed(host, name):
            minimal-rhel-7-hvm:         pkg = host.package(name)
            minimal-rhel-7-hvm:         if pkg.is_installed:
            minimal-rhel-7-hvm:             log.info(
            minimal-rhel-7-hvm:                 '%s',
            minimal-rhel-7-hvm:                 {'pkg': pkg.name, 'version': pkg.version, 'release': pkg.release})
            minimal-rhel-7-hvm: >       assert pkg.is_installed
            minimal-rhel-7-hvm: E       assert False
            minimal-rhel-7-hvm: E        +  where False = <package python36>.is_installed
            minimal-rhel-7-hvm:
            minimal-rhel-7-hvm: /tmp/spel/tests/test_ami.py:112: AssertionError
            minimal-rhel-7-hvm: ================ 1 failed, 22 passed, 1 skipped in 4.65 seconds ================
~~~

### Suggestions for fix

Update `python36` [test-block](https://github.com/plus3it/spel/blob/master/tests/test_ami.py#L103-L112) to do an OR-presence for `python3` such that, if _one_ is present, the test renders a success-state.

**Comment by ferricoxide** [comment]
Dispatching to @lorengordon since, presumably, as the author of the relevant test is most-familiar with appropriate fixes.

**Comment by ferricoxide** [comment]
Tried to see if there was a conditional could use to govern whether it tests for `python36` or `python3`. Only thing that looked like it might be usable would be a match against `platform.linux_distribution()[1]`

**Comment by lorengordon** [comment]
Closed by #327
